tests/parsing/errors/structure/expected/external.res.txt

```diff
@@ -0,0 +1,11 @@
+
+  Syntax error!
+  tests/parsing/errors/structure/external.res:2:51
+
+  1 │ // missing JS value name
+  2 │ external setTimeout: (unit => unit, int) => float =
+  3 │ 
+
+  An external requires the name of the JS value you're referring to, like "setTimeout".
```

Comment:
cc @bloodyowl, how does this sound? 😄

Reply:
Does this only occur with:

```
external setTimeout: (unit => unit, int) => float =
```

or also with

```
external setTimeout: (unit => unit, int) => float
```

?

Reply:
Currently only implemented for the first. Can also add one for the latter

Reply:
I'd go for:

```
2 | external setTimeout: (unit => unit, int) => float =
3 |
This external is missing the JavaScript value it is supposed to reference (like `= "setTimeout"`)
```

But the one is your PR looks good to me as well 😊